relentless/simulate/lammps.py

```diff
@@ -322,13 +323,31 @@ def pair_map(sim,pair):
                     for idx in range(Nr):
                         fw.write('{idx} {r} {u} {f}\n'.format(idx=idx+1,r=r[idx],u=u[idx],f=f[idx]))
 
+                    # find r where potential and force are zero
+                    nonzero_r = numpy.flatnonzero(numpy.logical_and(~numpy.isclose(u,0),~numpy.isclose(f,0)))
+                    if len(nonzero_r) > 1:
+                        # cutoff at last nonzero r (cannot be first r)
+                        rcut[(i,j)] = r[nonzero_r[-1]]
+                    else:
+                        # if first or second r is nonzero, cutoff at second
+                        rcut[(i,j)] = r[1]
+        else:
+            rcut = None
+            rcut = sim.communicator.bcast(rcut)
```

Comment:
This needs to be outside the `else` branch, as it must be called on all ranks
```suggestion
        rcut = sim.communicator.bcast(rcut)
```